extension/data.ts

```diff
@@ -940,60 +940,57 @@ class RcxDict {
   }
 
   makeText(entry: DictEntryData | null, max: number): string {
-    let e;
-    let i;
-    let j;
-    let t;
-
     if (entry === null) {
       return '';
     }
 
-    const b = [];
+    const result = [];
```

Comment:
In the long run, this type of structured output should probably be done with a template (such that the final text is easier to see in the code).

For HTML, I would naturally assume `lit-html` would be a good candidate since I use it for the options page.

However, I'm not sure if it would work for generating non-html text (haven't tried). If not maybe something like normal template literals would improve things (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) but we'd need to do something special for the list inputs...

Not a request to change anything but something I thought of when seeing your improvements!

Reply:
Yeah, that might make the code feel more declarative and easier to follow. I'll keep that in mind in the future.